src/h5web/visualizations/heatmap/DomainSlider.tsx

```diff
@@ -54,7 +47,12 @@ function DomainSlider(props: Props): ReactElement {
           </div>
         )}
         value={[...(value || dataDomain)]}
-        onChange={updateDomain}
+        onAfterChange={(bounds) => {
+          const roundedDomain = (bounds as number[]).map((val) =>
+            round(val, NB_DECIMALS)
+          ) as Domain;
+          onChange(roundedDomain);
+        }}
```

Comment:
`onAfterChange` is now called when the track is clicked. Since There is a built-in debounce, it means we can remove our own. 🎉

Reply:
Excellent !